src/Microsoft.Diagnostics.Monitoring.WebApi/SharedStrings.resx

```diff
@@ -0,0 +1,236 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
nit: rather than "SharedStrings", because there can be multiple sharing groups, maybe call this OptionsDisplayStrings to more clearly show the intent of what this particular resource file is used. I'd also like to avoid it becoming a dumping ground of strings that are shared between all of the assemblies.